apps/builder/app/builder/features/style-panel/style-source/style-source-control.tsx

```diff
@@ -241,6 +241,19 @@ const StyleSourceState = styled(Text, {
   },
 });
 
+const LocalStyle = styled(LocalStyleIcon, {
+  variants: {
+    hasStyles: {
+      true: {},
+      false: {
+        "& [fill]": {
```

Comment:
Too much reliance on icon internals. I would inline it and use css variable to hide circle.

Reply:
yeah, what you you do? add some explicit attribute?